src/__tests__/credentialManager.test.ts

```diff
@@ -0,0 +1,88 @@
+import { Configuration } from "../api/generated/configuration";
+
+import {
+    credentialManager,
+    CredentialStore
+} from "../api/credentialManager";
+
+const serverId1 = "f4486b851af24255b3305fe614b81f01";
+const serverConfig1: Configuration = {
+    apiKey: "b49268e51af24255b3305fe614b81f01"
+};
+
+const serverId2 = "af72hb851af24255b3305fe614b81f01";
+const serverConfig2: Configuration = {
+    apiKey: "d4286b8119f24a55b3305fe614b81f01"
+};
+
+describe("Getting servers from credentialManager", () => {
```

Comment:
I prefer all lowercase for tests, but since I'm not working on this project I won't actually request that change haha.